python/cudf/cudf/core/indexed_frame.py

```diff
@@ -1031,11 +1033,10 @@ def interpolate(
             if isinstance(data._index, cudf.RangeIndex)
             # TODO: This should be a scatter, avoiding an argsort.
```

Comment:
Is it easy to just make that fix to use a scatter here? Or are there new APIs/bindings that would be needed to do so?

Reply:
scatter doesn't exist at the frame level yet, so this would be a column-by-column scatter and then `_from_data`. I am inclined to do that separately.